sdk/appcenter-data/src/main/java/com/microsoft/appcenter/data/Utils.java

```diff
@@ -76,6 +78,33 @@ public synchronized Date deserialize(JsonElement json, Type typeOfT, JsonDeseria
         }
     }
 
+    private static <T> DocumentWrapper<T> parseLocalDocument(LocalDocument localDocument, Class<T> documentType) {
+        DocumentWrapper<T> documentWrapper = parseDocument(
+                localDocument.getDocument(),
+                localDocument.getPartition(),
+                localDocument.getDocumentId(),
+                localDocument.getETag(),
+                localDocument.getOperationTime(),
+                documentType);
+        documentWrapper.setFromCache(true);
+        documentWrapper.setPendingOperation(localDocument.getOperation());
+        return documentWrapper;
+    }
+
+    static <T> PaginatedDocuments<T> localDocumentsToNonExpiredPaginated(List<LocalDocument> localDocuments, Class<T> documentType) {
+        PaginatedDocuments<T> paginatedDocuments = new PaginatedDocuments<>();
+        Page<T> page = new Page<>();
+        ArrayList<DocumentWrapper<T>> documentWrappers = new ArrayList<>();
```

Comment:
Assign variable to the most generic collection type usually. Here you need only the Iterable interface.